### PR TITLE
kinflate: added version command

### PR DESCRIPTION
**Pull request by droot**
add version package. Will be using it in container builder to embed version in all the releases. It will help us during debugging any external issues.

### CODE REVIEW

File: cmd/kinflate/version/version.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	return Version{
+		kinflateVersion,
+		gitCommit,
+		buildDate,
+		goos,
+		goarch,
+	}
+}
+
+func (v Version) Print() {
```
Comment (mengqiy) [inline cmd/kinflate/version/version.go]: I'd like to see it prints to an `io.writer`.
IOW, `NewCmdVersion` takes another argument `stdOut io.writer`.

Comment (droot) [reply cmd/kinflate/version/version.go]: Done

**Comment by droot** [comment]
@mengqiy Thanks for the quick review. Addressed the comment about the writer.
Example output

```
Version: {KinflateVersion:1.6 GitCommit:$Format:%H$ BuildDate:1970-01-01T00:00:00Z GoOs:unknown GoArch:unknown}
```

In our releases, all the fields will be filled in properly.

**Pull request merged**